specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4242,16 +4246,20 @@
 		"properties": {
 			"name": {
 				"type": "string",
-				"description": "The name of the output."
+				"description": "The name path of the output."
 			},
 			"targetName": {
 				"type": "string",
 				"description": "The target name of the output."
 			}
 		},
-		"required": ["name",
-		"targetName"],
-		"description": "output field mapping for a skill."
+		"required": [
+		    "name", "targetName"
+		],
+		"externalDocs": {
+			"url": "https://docs.microsoft.com/rest/api/searchservice/naming-rules"
+	    },
+		"description": "Output field mapping for a skill."
```

Comment:
Name of the skill output.

Reply:
Synced up with Victor offline on this. I will need to add the OutputFieldMappings field into the indexer.